app/javascript/gobierto_data/webapp/components/sidebar/SidebarDatasets.vue

```diff
@@ -111,6 +95,27 @@ export default {
       }
     }
   },
+  watch: {
+    currentDatasetSlug: function(newSlug) {
+      this.showLess = true;
+      if (this.sortedItems.length && newSlug !== '') {
```

Comment:
If you only wanna check if `newSlug` has value, leave this:
```suggestion
      if (this.sortedItems.length && newSlug) {
```